src/main/java/org/interledger/ilp/ledger/LedgerAdaptor.java

```diff
@@ -66,7 +66,7 @@
    * @param transfer The transfer being proposed.
    * @param reason The reason why the transfer should be rejected.
    */
-  void rejectTransfer(LedgerTransfer transfer, TransferRejectedReason reason);
+  void rejectTransfer(LedgerTransfer transfer, InterledgerException.ErrorCode reason);
```

Comment:
@adrianhopebailie Thanks for the review. I just updated the code. 
- Regarding the ErrorType it's exposed now as a (enum) property of the error throught the getErrorType, but at Exception-creation-time it's the first character of errorCode that will used/parsed to set the type since developers will have to indicate the errorCode and forcing them to also indicate the errorType will just cause them to set duplicated information (with marginal performance benefits) and the possibility of coding errors.

Let me know if you have a different point of view.

- Note the TODO:(RFC) indicates some point that could be better to fix (or better explain) on the RFC.